assets/stylesheets/components/_headings.scss

```diff
@@ -46,10 +46,20 @@ h6 {
   }
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   .dark-theme & {
     $headings-color: $cr-gray-dark;
 
     color: $headings-color;
   }
+
+  a,
+  a & {
+    color: $cr-gray-dark;
```

Comment:
@seancdavis I'm apprehensive about tying `<a>` nested within header tags to this specific color. My first thought is that `$cr-gray-dark` appears to only apply to the headers if a dark theme is enabled, so the `<a>` being `$cr-gray-dark` all the time might not make the most sense. Another idea would be to declare the embedded `<a>` tags with a `color: inherit` rule to keep us from having to play wack-a-mole trying to figure out all of the color permutations for embedded links. Thoughts?